pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+pythonpath = hire_hire
+DJANGO_SETTINGS_MODULE = hire_hire.settings
+python_files = test_*.py
+addopts = -p no:cacheprovider
+django_debug_mode = keep
+filterwarnings =
+    ignore::DeprecationWarning
+    ignore::django.utils.deprecation.RemovedInDjango51Warning
```

Comment:
add new line to end of the file
![image](https://github.com/hire-hire/hire-hire/assets/91815912/f778a99f-250a-43ed-b751-300add4ba2b3)